CustomScript/HandlerManifest.json

```diff
@@ -1,13 +1,17 @@
-[{
-  "name": "CustomScript",
-  "version": "1.5.1.0",
-  "handlerManifest": {
-    "installCommand": "customscript.py -install",
-    "uninstallCommand": "customscript.py -uninstall",
-    "updateCommand": "customscript.py -update",
-    "enableCommand": "customscript.py -enable",
-    "disableCommand": "customscript.py -disable",
-    "rebootAfterInstall": false,
-    "reportHeartbeat": false
+[
+  {
+    "namespace": "Microsoft.OSTCExtensions",
+    "name": "CustomScriptForLinux",
+    "extension_version": "1.5.2.0",
```

Comment:
> Any objections to moving them to manifest.xml? They have to be there anyway.

They are already in manifest.xml as you said, so it would be better to not to have redundancy here. You can refer to the pdf I pointed out earlier for the precise schema.